SegmentEditorFloodFilling/SegmentEditorFloodFillingLib/SegmentEditorEffect.py

```diff
@@ -115,6 +116,7 @@ def masterVolumeNodeChanged(self):
   def setMRMLDefaults(self):
     self.scriptedEffect.setParameterDefault("IntensityTolerance", 10.0)
     self.scriptedEffect.setParameterDefault("NeighborhoodSizeMm", 1.0)
+    self.scriptedEffect.parameterSetNode().SetNodeReferenceID("ROINodeID", None)
```

Comment:
To be consistent with existing naming convention (e.g., in SegmentEditorMaskVolumeLib\SegmentEditorEffect.py), the role name has to be `Flood Filling.ROI`